Add total kills and session kills

PJGJ210 opened the pull request:
-Update runelite version
-Add actual total kills (updates when checking bow)
-Change GUI text for session versus total kills
-Opens GUI when checking total kills

PJGJ210 committed:
-Update runelite version
-Add actual total kills (updates when checking bow)
-Change GUI text for session versus total kills
-Opens GUI when checking total kills

PJGJ210 commented:
![image](https://user-images.githubusercontent.com/16944807/181273135-b355cc6e-01ce-4d8c-b82d-a187b15452a2.png)

PJGJ210 committed:
Delete vcs.xml

PJGJ210 committed:
Delete .gitignore

PJGJ210 committed:
Delete compiler.xml

PJGJ210 committed:
Delete gradle.xml

PJGJ210 committed:
Delete jarRepositories.xml

PJGJ210 committed:
Delete misc.xml

PJGJ210 committed:
Delete build.gradle

PJGJ210 committed:
-Add actual total kills (updates when checking bow)
-Change GUI text for session versus total kills
-Opens GUI when checking total kills

PJGJ210 committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	.idea/gradle.xml
#	build.gradle

Lastrick1 commented:
> ![image](https://user-images.githubusercontent.com/16944807/181273135-b355cc6e-01ce-4d8c-b82d-a187b15452a2.png)

I'm relatively new to github, so please excuse me if this is in the wrong location. I was hoping to write my own RL plugins and modifications, including one for Chompy and I was wondering how you test your modifications. Put another way, how would I test my new/additional code for the Chompy plugin (e.g., timers for the toad or total bellows remaining in inventory) without it violating Jagex policy? Thanks.

PJGJ210 commented:
Not exactly the place. Below is rules for plug-ins. I do testing on live
server and I would guess others do too.

https://secure.runescape.com/m=news/another-message-about-unofficial-clients?oldschool=1

For dev help on new plugins see below.

https://github.com/runelite/plugin-hub/blob/master/README.md

Just a note on ejedev though, they appear active in git but not chompy/rs.
I was tempted to repost/remake this plug-in (as pulls have sat here for a
while) but have not done so.

On Thu, Aug 25, 2022, 5:04 PM Andy Strickland ***@***.***>
wrote:

> [image: image]
> <https://user-images.githubusercontent.com/16944807/181273135-b355cc6e-01ce-4d8c-b82d-a187b15452a2.png>
>
> I'm relatively new to github, so please excuse me if this is in the wrong
> location. I was hoping to write my own RL plugins and modifications,
> including one for Chompy and I was wondering how you test your
> modifications. Put another way, how would I test my new/additional code for
> the Chompy plugin (e.g., timers for the toad or total bellows remaining in
> inventory) without it violating Jagex policy? Thanks.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ejedev/chompy-hunter/pull/5#issuecomment-1227760727>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEBI5JZHL3ILDS3AMWO6PQDV27NUJANCNFSM54YBFQXQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

PJGJ210 committed:
-Updated to include Redux in title
-Added a preferred width to info overlay
-Added a few tags
-Included updates from commit 05aeed21c52c52f994a26e43a4f7c68eb828d87c to remove 15 second green highlight.

PJGJ210 committed:
Update README.md

PJGJ210 committed:
Update README.md

PJGJ210 committed:
Update Chompy Hunter Plugin
-Added a preferred width to info overlay
-Added a few tags
-Included updates from commit 05aeed21c52c52f994a26e43a4f7c68eb828d87c to remove 15 second green highlight.

PJGJ210 committed:
Merge remote-tracking branch 'origin/master'

PJGJ210 committed:
Update README.md

PJGJ210 committed:
Update README.md

PJGJ210 committed:
Update README.md

PJGJ210 committed:
Update Chompy Hunter Plugin
-Set to enable by default

PJGJ210 committed:
Merge remote-tracking branch 'origin/master'

PJGJ210 committed:
-Added a timeout option based on region

PJGJ210 committed:
-Remove loadWhenOutdated

PJGJ210 committed:
Update README.md

Lucidare committed:
Added .gitignore, updated license to include self, updated runelite version in build.gradle to be latest.release, updated version to 2.0, fixed issue where dead chompies were notified, fixed issue where timers were lost when chompy despawns off screen, added Lucid Poro to authors

Lucidare committed:
Updated to use NpcIds

PJGJ210 committed:
Merge pull request #7 from PJGJ210/bug-fixes

Various Bug Fixes + Added Contribution

PJGJ210 committed:
-Add option to hide the information panel
